app/viewmodels/govuk/RadiosFluency.scala

```diff
@@ -112,6 +112,32 @@ trait RadiosFluency {
         items = items
       ).inline()
     }
+
+    def yesNoWithCustomValues(
+                               field: Field,
+                               legend: Legend,
+                               yesValue: String,
+                               noValue: String
+                             )(implicit messages: Messages): Radios = {
+      val items = Seq(
+        RadioItem(
+          id      = Some(field.id),
```

Comment:
Is it worth following a specific style here for indentation? This file has two diffrent styles...

Reply:
Resolved in [065c553](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/19/commits/065c5537a628da73e3937d89778dc7136d7681af)